plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/PomData.java

```diff
@@ -43,12 +43,14 @@
     @CheckForNull
     public final MavenCoordinates parent;
     private String connectionUrl;
+    private String scmTag;
     private List<String> warningMessages = new ArrayList<String>();
 
-    public PomData(String artifactId, @CheckForNull String packaging, String connectionUrl, @CheckForNull MavenCoordinates parent){
+    public PomData(String artifactId, @CheckForNull String packaging, String connectionUrl, String scmTag, @CheckForNull MavenCoordinates parent){
```

Comment:
Would be nice to retain the binary compatibility if possible. It is a library part though I am not sure whether anybody really uses it as a lib

Reply:
[These](https://www.google.com/search?q=%22PomData%22+site%3Agithub.com%2Fjenkinsci&oq=%22PomData%22+site%3Agithub.com%2Fjenkinsci) [two](https://www.google.com/search?q=%22org.jenkins.tools.test.model.PomData%22) Google searches don't find any usages outside of `plugin-compat-tester`. Preserving binary compatibility for a class that's only used in this project seems overkill to me at first glance, but I can add a new constructor if you feel strongly about this.

Reply:
I do not believe code in in this repository is statically linked to from anything else, so it may be freely refactored.

Reply:
We just broke the binary compatibility in 0.2.0, so I retract my comment  here